client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -0,0 +1,208 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Sidebar from 'layout/sidebar';
+import SidebarFooter from 'layout/sidebar/footer';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarRegion from 'layout/sidebar/region';
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class JetpackCloudSidebar extends Component {
+	static propTypes = {
+		context: PropTypes.shape( {
+			path: PropTypes.string,
```

Comment:
Also, should this prop be required? Seems like usage is assuming it will always be defined, so maybe let's reflect that in the proptypes definition.

Reply:
Sounds good 👍

Reply:
Addressed in 603c49e.